easybuild/easyconfigs/d/Dakota/Dakota-6.11.0-foss-2021b.eb

```diff
@@ -0,0 +1,63 @@
+# /!\ IMPORTANT: you need at least 8 cores to complete successfully the tests
+
+easyblock = "CMakeMake"
+
+name = "Dakota"
+version = "6.11.0"
+
+homepage = "https://dakota.sandia.gov/"
+description = """
+ The Dakota project delivers both state-of-the-art research and robust, usable software for optimization and UQ.
+ Broadly, the Dakota software's advanced parametric analyses enable design exploration, model calibration,
+ risk analysis, and quantification of margins and uncertainty with computational models."
+"""
+
+toolchain = {"name": "foss", "version": "2021b"}
+# Disable architecture optimization, otherwise the basic tests fail
+toolchainopts = {
+    "pic": True,
+    "usempi": True,
+    "optarch": False,
+    "extra_fflags": "-fallow-argument-mismatch",
+}
+
+source_urls = ["https://dakota.sandia.gov/sites/default/files/distributions/public/"]
+sources = ["dakota-%(version_major_minor)s-release-public.src.tar.gz"]
+patches = ["Dakota_fix_boost.patch"]
+checksums = [
+    'd38bbfccba4ff5f2187bdbb45633e269ef4d9f68631a17a52b4d05b5f9b75f65',  # dakota-6.11-release-public.src.tar.gz
+    'a0cd19bdd860976cb97db845a628a0807316a8156f0339894790c601ba1a15cd',  # Dakota_fix_boost.patch
+]
+
+dependencies = [
+    ("Boost", "1.79.0"),
```

Comment:
does Boost 1.77.0 work to here?
that would make the failing test go away.
if not, you'll have to add a Boost versionsuffix.